### ISSUE TITLE
`A.uniq`: hasOwnProperty is not a function

### ISSUE THREAD

**Issue by mmiszy** [mobily/ts-belt#10]
I'm using `A.uniq` on an array of Tuples. Unfortunately, that results in an exception being thrown:

```
@mobily/ts-belt/dist/cjs/Array/index.js:446
        if (!b.hasOwnProperty(key)) {
               ^
TypeError: b.hasOwnProperty is not a function
```

**Comment by mmiszy** [comment]
I guess you should be using `Object.prototype.hasOwnProperty.call(b, key)` instead.

**Comment by mobily** [comment]
@mmiszy 👋 could you provide a simple code example, please?

**Comment by mmiszy** [comment]
Sure: https://runkit.com/embed/c5nesq8njosj

**Comment by mobily** [comment]
`Object.prototype.hasOwnProperty.call(b, key)` this did a trick, thanks! fixed and published in `v3.4.0` 🚀
